installer/scripts/omsadmin.sh

```diff
@@ -427,6 +433,7 @@ onboard()
     
     if [ $error -ne 0 ]; then
         log_error "Error during the onboarding request. Check the correctness of the workspace ID and shared key or run omsadmin.sh with '-v'"
+        rm "$FILE_CRT" "$FILE_KEY" > /dev/null 2>&1 || true
```

Comment:
Please note that, it's not the renew certificate scenario, where we will backup the old certificate, and overwritten when the renew succeed.